src/interpret/lianapy/config.vsh.yaml

```diff
@@ -0,0 +1,85 @@
+functionality:
+  name: lianapy
+  namespace: "interpret"
+  description: "Performs LIANA integration based as described in https://github.com/saezlab/liana-py"
+  arguments:
+    - name: "--input"
+      alternatives: ["-i"]
+      type: file
+      description: Input h5mu file
+      direction: input
+      required: true
+    - name: "--output"
+      alternatives: ["-o"]
+      type: file
+      description: Output h5mu file.
+      direction: output
+      required: true
+    - name: "--output_compression"
+      type: string
+      choices: ["gzip", "lzf"]
+      default: "gzip"
+      required: false
+    - name: "--modality"
+      type: string
+      default: "rna"
+      required: false
+    - name: "--layer"
+      type: string
+      default: "None"
```

Comment:
by default, viash will convert yaml's `null` (the default for the property `default:`) in the config yaml to the appropriate corresponding type in the scripts. For python, this is `None`. So you can just remove this `default: "None"` line (or use `default: null`), and remove the 

```python3
    if par['layer'] == "None":
        par['layer'] = None
```

from the script. 

Also, my previous comment was in regards to the description, not the behavior of the parameter:

```
"Layer in anndata.AnnData.layers to use. If None, use anndata.AnnData.X."
```

to
```
"Layer in anndata.AnnData.layers to use. If None, use mudata.mod[modality].X."
```

I prefer to use the `X` layer over `log_normalized`, so also keep that change.

Reply:
Got it. I made the changes, hope it's better now.

Reply:
Looks good!